packages/neos-ui-sagas/src/UI/ContentTree/index.js

```diff
@@ -142,7 +142,7 @@ export function * watchCurrentDocument({globalRegistry, configuration}) {
         }
 
         const state = yield select();
-        const childrenAreFullyLoaded = $get(['cr', 'nodes', 'byContextPath', contextPath, 'children'], state)
+        const childrenAreFullyLoaded = ($get(['cr', 'nodes', 'byContextPath', contextPath, 'children'], state) || [])
```

Comment:
this fix could be backported as well no?

(would also take care of it just vote: 👍  👎 )

Reply:
My plan was to leave the plow-js calls as they are for now and take care of replacing them in https://github.com/neos/neos-ui/pull/3306, right after the E2E tests have been merged.